src/components/AudioPlayer/RepeatAudioModal/SelectType.tsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from 'react';
+
+import Combobox from 'src/components/dls/Forms/Combobox';
+import RadioGroup, { RadioGroupOrientation } from 'src/components/dls/Forms/RadioGroup/RadioGroup';
+import { RangeSelectorType } from 'src/components/Verse/AdvancedCopy/SelectorContainer';
+import VerseRangeSelector from 'src/components/Verse/AdvancedCopy/VersesRangeSelector';
+
+export enum RepeatType {
+  Single = 'single',
+  Range = 'range',
+}
+
+const repeatTypeRadioGroupItems = [
+  {
+    value: RepeatType.Single,
+    id: RepeatType.Single,
+    label: 'Single Verse',
+  },
+  {
+    value: RepeatType.Range,
+    id: RepeatType.Range,
+    label: 'Range of verses',
+  },
+];
+
+const SelectType = ({
```

Comment:
Type generally indicates a TS Type, we should use a different name for the component.